tests/integ/modin/index/test_index_methods.py

```diff
@@ -11,6 +11,8 @@
 from tests.integ.modin.sql_counter import SqlCounter, sql_count_checker
 from tests.integ.modin.utils import assert_index_equal, assert_series_equal
 
+data = {"col1": [1, 2, 3], "col2": [3, 4, 5], "col3": [5, 6, 7]}
```

Comment:
I would actually define this as a pytest.fixture instead and pass it in as a parameter to the functions where it is used

Reply:
agree, can you define an empty index for all those tests too?

Reply:
there is an empty index in the native test data